src/main/resources/application.yaml

```diff
@@ -5,7 +5,7 @@ spring:
     hibernate:
       ddl-auto: none
   datasource:
-    url: jdbc:postgresql://localhost:8081/userapi
+    url: jdbc:postgresql://localhost:8081/userapi_app
```

Comment:
Why did you change the database name?

Reply:
There was a problem with the connection with it. The database name was incorrect, that's why I changed it.